toml.md

```diff
@@ -545,6 +544,9 @@ sf5 = +nan # same as `nan`
 sf6 = -nan # valid, actual encoding is implementation-specific
 ```
 
+Implementations are free to support any precision level. It's recommended that
+at least IEEE 754 binary64 values are accepted.
```

Comment:
I would add the "and handled losslessly" from above here too. Otherwise the "accepted" would be pretty meaningless, since it could be interpreted as "accepted, but internally converted to a lower precision".

Reply:
But then we're back to your [original comment](https://github.com/toml-lang/toml/pull/1058#issuecomment-2823653871) that "handled losslessly" doesn't really work for IEEE floats?

Reply:
Hmm, well, the problem already lies in the "accepted". What's accepted, i.e. parsed, is originally never a float, but rather a decimal written as string. So I guess what we mean is not that they are "accepted", but they there are used internally, meaning that any decimals will be parsed with sufficient precision.

So how about changing it to "It's recommended that at least IEEE 754 binary64 values are *supported*" ? That should do the trick, I think.

Reply:
@arp242 So what do you think about changing "accepted" to "supported" to finalize this?